stable/openstack-base/bundle.yaml

```diff
@@ -1,35 +1,36 @@
+local_overlay_enabled: true
+series: focal
+# *** Please refer to the OpenStack Charms Deployment Guide for more        ***
+# *** information.
+# *** https://docs.openstack.org/project-deploy-guide/charm-deployment-guide **
+# ***                                                                       ***
+# *** openstack-origin should be changed back to distro once this PPA is    ***
+# *** made redundant.                                                       ***
+variables:
```

Comment:
I had thought we were doing away with the variables section?

Reply:
Since we don't really have a good alternative and have solved the problem with unsolicited expansion on update of versions I kept them.